### ISSUE TITLE
@mixin column() could be more flexible in breakpoint definitions.

### ISSUE THREAD

**Issue by obrogaming** [connectivedx/Phoenix#81]
Be nice if given a global variable within /variables/_grid.scss that would allow us to define the responsive breakpoint value used @mixin column() (/mixins/grid/_semantic-grid.scss) that is currently a fixed value instead. Might even be better to allow us to also pass a breakpoint value to @mixin column(), while also having a global breakpoint fall-back variable as well.

Seems very fixed right now, and could easily extend some interesting flexibility to the development.

**Comment by ajmueller** [comment]
Now that we have Sass map support as of release 1.0.2, this is something that we can begin to explore more.  I've also marked this as a dependent since it will rely on the changes made in #105, which have been merged into the development branch.  Some initial thoughts here:

- What do we define at each breakpoint?  Just width via column count?  How about padding, alignment, and the shifted and nested properties?  Assuming the simplest initial implementation, we would just have column count.  The remaining points below assume this.
- Column widths should be defined as a map with this structure:

    ```scss
    $columns: (
        (
            breakpoint: breakpointName,
            columns: columnCount
        ),
        (
            breakpoint: breakpointName,
            columns: columnCount
        )
    );
    ```
    This map would need to be passed into the mixin.  We currently have two parameters, `$columns` and `$options`, and this new map should replace `$columns` as the first parameter.  It should be a required parameter and since we currently define a default breakpoint, should we still provide a default or opt for it to be up to the user to explicitly express their breakpoint choice(s)?

Overall I think there's a lot of potential here to make an (I'm assuming) seldom-used mixin much more useful.  However, we should also ask if this is a worthwhile exploration considering the potential of #74 and [CSS grids](http://css-tricks.com/snippets/css/complete-guide-grid/) (which appear to be a ways off).  Perhaps the mixin will become even more useful as flexbox and grid become the norm and the mixin will be a means of providing a meaningful fallback to older browsers.
